doc/menu_lua_api.txt

```diff
@@ -192,6 +192,9 @@ core.settings:set_bool(name, value)
 core.settings:get_bool(name) -> bool or nil (possible in async calls)
 core.settings:save() -> nil, save all settings to config file
 
+For a more comprehensive list of methods of the Settings object see
+[lua_api.txt](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L4759)
```

Comment:
That's only for Markdown. This is in plain text.

Just drop the title and parentheses. Leave the link as is.

Reply:
`lua_api.txt` and `client_lua_api.txt` are both also text files, but written so they can be read readily with a markdown application. This file was not changed to that format but markdown links seem acceptable here too.

Reply:
I agree with @SmallJoker here. All the `*lua_api.txt` files are in markdown format, but have the `.txt` extension for easy linking to specific lines. e.g. https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L128

Reply:
Yes MD format is fine here.

Reply:
Ok, got it. Sorry for wasting your time!